docker-compose.yml

```diff
@@ -1,9 +1,14 @@
 version: '3'
 services:
   odca_search_engine:
-    image: odca-search-engine
+    image: olichwiruk/odca-search-engine:v0.2.0
```

Comment:
let's change that right away to https://hub.docker.com/repository/docker/humancolossus/oca-repository

Reply:
changed